src/app/shared/navbar/navbar.html

```diff
@@ -1,19 +1,22 @@
 <!-- TODO: figure out if the <nav> should go inside of a <header> element. -->
-<nav class="docs-navbar">
+<nav class="docs-header">
   <a md-button class="docs-button" routerLink="/" aria-label="Angular Material">
-    <img class="docs-angular-logo"
-      src="../../../assets/img/homepage/angular-white-transparent.svg"
-      alt="angular">
+    <md-icon class="angular-logo" svgIcon="angular"></md-icon>
```

Comment:
Any reason to make this change? Switch to `md-icon` does incur a cost, and we don't need to have an SVG here since we don't style the Angular logo

(same for github logo)

Reply:
I think it should be okay to keep using `img`; I want to get better about encouraging people to only use `md-icon` when there'd be some advantage in doing so